docs/grading/status_updates.md

```diff
@@ -24,18 +24,16 @@ We ask that each Status Update comprise the following parts. **Each part must be
 State your name and project.
 
 #### 1: This Week
-Summarize what you have contributed in the past week. This includes code, documentation, learning, etc. Remember that someone reading might not be familiar with your project, so give context. If you're unsure how much detail to give, lean on the more detailed side.
+Summarize what you have contributed in the past week. This includes code, documentation, learning, etc. Remember that someone reading might not be familiar with your project, so **give context**. If you're unsure how much detail to give, lean on the more detailed side.
 
 #### 2: Next Week
-Summarize what you plan on doing next week. We aren't holding you to this, but we want you to at least think about it each week.
+Summarize what you plan on doing next week. We want you to at least think about where your project is going, and what that might require of you.
 
 #### 3: Blockers
-List any *technical* blockers you have hit when trying to work on your project. This **does not** include things outside of RCOS that prevented you from getting work done, i.e. "I had a test this week so I couldn't do work"
-
+List any *technical* blockers you have hit when trying to work on your project. This **does not** include things outside of RCOS that prevented you from getting work done, i.e. "I had a test this week so I couldn't do work." Students with blockers are encouraged to reach out to their project leads or small group mentors so they can continue development.
 
 #### 4: Links
-List direct links to your contributions and any resources you were using for learning. Mentors need to be able to see proof of your work by following these links. Links to your commits, issues, etc. are desirable.
-
+List direct links to your contributions. Mentors need to be able to see proof of your work by following these links. Links to your commits, issues, etc. are desirable.
```

Comment:
add that contributions can include learning summaries, following tutorials on a sample repo, etc.